website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Jira Cloud Integration - Installation
+---
+
+## Prerequisites
+
+### Jira Cloud
+
+- [Jira **Cloud**](https://www.atlassian.com/software/jira)
+
+For Jira Data Center, [see the Jira Server plugin](/integrations/jira_server_plugin_installation)
+
+### Unleash
+
+- Unleash v4 or higher
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash admin to configure an admin API token. (Unleash Open Source)
+
+For Unleash Enterprise we recommend [Service Tokens](https://docs.getunleash.io/how-to/how-to-create-service-accounts), this is also a requirement for integrating with Change Requests
+
+### Jira
+
+You will need a Jira admin user.
+
+## Installation
+
+If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+
+> More to be added here after we can have screenshots and guides of the installation procedure
+
+## Configuring the plugin {#configure-plugin}
+
+After the plugin has been installed, each project's settings page in Jira will have a menu entry link called "Unleash Project Settings" under the "Apps" menu section.
+
+![Jira Cloud - Project settings link](/img/jira_cloud_project_settings_link.png)
```

Comment:
Can we improve this alt text? Without seeing this image, I have no idea what it is supposed to be. For people with vision impairments or with slow enough networks that images are impractical: what should they get out of the image?

The same goes for the rest of the alt texts in this PR.